CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ## 2.4.1 (Unreleased)
```

Comment:
I believe the version needs to be updated from 2.4.1 to 2.5.0 since we are adding new functionality following guidelines in XEP-7 and semantic versioning we are trying to follow for Alfred API